pydisdrometer/tests/test_plot.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import unittest
+
+from ..plot import plot
+
+
+class TestPlot(unittest.TestCase):
+    'Test module for the plot scripts'
+
+    def setUp(self):
+        filename = 'testdata/sgpdisdrometerC1.b1.20110427.000000_test_jwd_b1.cdf'
+        self.dsd = ARM_JWD_Reader.read_arm_jwd_b1(filename)
```

Comment:
This probably needs an import above. Also we'll need to modify the travis runner to be able to handle matlab plots.